extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -104,6 +117,7 @@ export class AmpStoryInteractiveSlider extends AmpStoryInteractive {
     super.initializeListeners_();
 
     this.inputEl_.addEventListener('input', () => {
+      this.isInteracting_ = true;
```

Comment:
We can use `cancelAnimationFrame(this.currentRAF_);` to cancel the animation.
Then we can delete the `this.isInteracting_` variable and the check on line 177.